register_user.sh

```diff
@@ -20,10 +20,10 @@ cd "$( dirname "$0" )" || { echo 'cd error' >&2; exit 1; }
 if [ ! -e app/config.ini ]
 then
     cat <<-'END_ERROR' >&2
-	Couldn't find config file 'app/config.ini'
-	Create this file from 'app/config.ini.default' with the credentials
-	to your database and then run this script again.
-	END_ERROR
+    Couldn't find config file 'app/config.ini'
+    Create this file from 'app/config.ini.default' with the credentials
+    to your database and then run this script again.
+    END_ERROR
```

Comment:
Oh but this won't work, you need tabs for the `<<-`-heredocs to be indented! https://unix.stackexchange.com/questions/76481/cant-indent-heredoc-to-match-nestings-indent